components/sup/src/census.rs

```diff
@@ -844,7 +839,7 @@ mod tests {
     /// Create a bare-minimum CensusMember with the given Health
     fn test_census_member(id: &str, health: Health) -> CensusMember {
         CensusMember { member_id: id.into(),
-                       pkg: None,
+                       pkg: PackageIdent::default(),
```

Comment:
That is terrifying I will be sure to avoid `Default` like the plague. Addressed in [b1f90d](https://github.com/habitat-sh/habitat/pull/7455/commits/b1f90dac1e778ce91d1731724399297f96f4ee91)

Reply:
`Default` is great... but only if there actually is a legitimate default value!